ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -317,27 +239,28 @@ def create_instance_in_clusterlogging(sc_name=None):
     else:
         logger.error("Instance for clusterlogging is not created properly")
 
-    pv_obj = ocp.OCP(
+    pod_obj = ocp.OCP(
         kind=constants.POD, namespace='openshift-logging'
     )
-    ret = pv_obj.wait_for_resource(condition=constants.STATUS_RUNNING,
-                                   resource_count=11,
-                                   timeout=150, sleep=5
-                                   )
-    assert ret, "Pods are not in Running state."
+    pod = pod_obj.wait_for_resource(
+        condition=constants.STATUS_RUNNING, resource_count=11, timeout=200,
+        sleep=5
+    )
+    assert pod, "Pods are not in Running state."
     logger.info("All pods are in Running state")
-    pv_obj = ocp.OCP(
+    pvc_obj = ocp.OCP(
         kind=constants.PVC, namespace='openshift-logging'
     )
-    ret = pv_obj.wait_for_resource(condition=constants.STATUS_BOUND,
-                                   resource_count=3, timeout=150, sleep=5
-                                   )
-    assert ret, "PVCs are not in bound state."
+    pvc = pvc_obj.wait_for_resource(
```

Comment:
pvc should be replaced as well - this is the status returned from wait_for_resource

Reply:
Done